team.html

```diff
@@ -26,6 +29,11 @@ <h1>Team</h1>
     <img alt="University of Arizona logo" class="teamlogo" src="/images/logos/ua_horiz_rgb_small.png">
   </div>
   <div class="teamlogo-wrapper">
-    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/eere_logo_horiz_green_small.png">
+    <img alt="PV Pros logo" class="teamlogo" src="/images/logos/PVPros-logo-340x93-1.png">
+  </div>
+  <div class="teamlogo-wrapper">
+    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/bay4-logo.png">
   </div>
-</div
+  <div class="teamlogo-wrapper">
+    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/eere_logo_horiz_green_small.png">
+  </div></div
```

Comment:
looks like this last `</div>` tag is missing the closing `>` in both the original file and this PR.